wikipendium/wiki/models.py

```diff
@@ -135,6 +135,8 @@ def clean(self):
         if '/' in self.title:
             raise ValidationError('Title cannot contain slashes')
 
+        self.content.replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
So you think the bug might not be fixed yet?

I honestly don't remember if we went over all previously stored compendiums and converted them, but if you have found a case where this is still broken we should look into it again.